core/common/src/main/java/alluxio/Constants.java

```diff
@@ -139,6 +139,8 @@
   public static final String LOCALITY_RACK = "rack";
   public static final String MESOS_LOCAL_INSTALL = "LOCAL";
 
+  public static final String SKIP_CONF_VALIDATION = "alluxio.skip.conf.validation";
```

Comment:
it can be implemented as a property key. I was choosing between the env variable or the property key. I don't have a strong preference. Will make this a property then if you think it is better.